ansible/scripts/cluster.sh

```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-inventory=${INVENTORY:-inventory}
+inventory=${INVENTORY:-../inventory}
 
-ansible-playbook -i ${inventory} cluster.yml $@
+ansible-playbook -i ${inventory} ../playbooks/cluster.yml $@
```

Comment:
I don't think we are ever going to create inventory directory with anything else than inventory examples. So we should be fine here.